cmd/soroban-cli/src/commands/network/container/logs.rs

```diff
@@ -23,9 +26,10 @@ pub struct Cmd {
 }
 
 impl Cmd {
-    pub async fn run(&self) -> Result<(), Error> {
+    pub async fn run(&self, global_args: &global::Args) -> Result<(), Error> {
+        let printer = print::Print::new(global_args.quiet);
```

Comment:
Can we name this `print` instead, because I've seen that's how you named it in another file (`cmd/soroban-cli/src/commands/network/container/stop.rs`).

I think `print` was also the name I've used in other places.

Reply:
Yeah for sure! I'll get a change pushed up shortly.